lib/dry/view/exposure.rb

```diff
@@ -23,38 +24,71 @@ def bind(obj)
         self.class.new(name, proc, obj, options)
       end
 
-      def dependencies
-        proc ? proc.parameters.map(&:last) : []
+      def dependency_names
+        if proc
```

Comment:
Everything looks great much clean and understandable.

I'm wondering if it makes sense to extract the behavior of `dependency_names` and `input_keys` since they are the same method but only changing the constant.

What would you do @timriley? Anyway, this is not a blocking change we can surely merge this one 🎉

Reply:
Hmm, you're right that they're same, but I think "DRY-ing" this up would probably just make these harder to understand, given there's only 2 instances of methods like this. So perhaps let's leave them for now, and reconsider this if we find ourselves wanting to add a 3rd such thing?